content/02.introduction.md

```diff
@@ -130,6 +147,17 @@ The results of COVID-19 infection can vary greatly among pediatric patients as w
 In particular, some children have experienced a severe inflammatory syndrome after COVID-19 infection, similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069;@doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117].
 Heart failure can also occur [@doi:10.1161/CIRCULATIONAHA.120.048360].
 
+Genetic factors may also play a role in risk of respiratory failure for COVID-19 [@doi:10.1056/NEJMoa2020283; @doi:10.1093/gerona/glaa131; @doi:10.1101/2020.06.16.155101], including possibly higher risk to individuals of blood group A and possibly lower risk to individuals of type O [@doi:10.1056/NEJMoa2020283].
+
+Diabetes may increase the risk of lengthy hospitalization [@doi:10.1016/j.cmet.2020.04.021] or of death [@doi:10.1016/j.cmet.2020.04.021;@doi:10.1007/s00592-020-01546-0].  
+[@doi:10.1111/dom.14057] and [@doi:10.1152/ajpendo.00124.2020] discuss possible ways in which COVID-19 and diabetes may interact.  
+On a speculative basis, the mechanisms described in a report by [@doi:10.1007/s00592-009-0109-4] involving effects on the pancreas by the related SARS-CoV-1 virus, which could reportedly cause short-term diabetic symptoms, might perhaps be relevant here as well.
+
+Obesity also appears to be associated with higher risk of severe outcomes from SARS-CoV-2, [@doi:10.1016/j.metabol.2020.154262; @doi:10.1101/2020.04.23.20076042] although 
+further research is needed regarding the exact mechanisms [@doi:10.1016/j.medj.2020.06.005].
+
+Because different subpopulations may have somewhat different vulnerabilities, needs, and resources, we recommend that researchers publishing studies on diagnostics and therapeutics take extra care to be clear about the demographic and medical characteristics of their sample, in order to facilitate discussions of the degree to which results may generalize or differ in other settings.
```

Comment:
That last sentence might be useful for a conclusion section if we have one @rando2

Reply:
Talked to Casey -- we will add a conclusion but probably not for a week or two since anything could change by then!